docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`5455` ETH2 staking rewards are now taken into consideration during accounting.
```

Comment:
Hmm it is kinda mixed here already, but if I am not mistaken, we tend to keep the list of features on top and the list of bugs in the bottom

Reply:
just fyi the order here does not really matter. It's just for us to look nice. It's rendered differently in the actual changelog: https://rotki.readthedocs.io/en/latest/changelog.html